app/misc/authenticate.ts

```diff
@@ -139,7 +139,7 @@ function getUserInfo(callback) {
           $("#otpModal").modal('show');
         });
       }
-      else if (err == "Error: getaddrinfo ENOTFOUND api.github.com api.github.com:443"){
+      else if (err.toString().indexOf("ENOTFOUND") != 0 || err.toString().indexOf("ENOENT") != 0){
         displayModal("No internet connection");
```

Comment:
Can we update this dialog to be more specific? Perhaps "Unable to Authenticate: No internet Connection" -- assuming that this is the ONLY case that would see this dialog?

Reply:
From what I've read on this error message it's specfically a DNS thing so how about "DNS not found: Check your internet connection"?

Reply:
It seems like ENOENT is a bit more generic actually, it just means there is nothing there. Do we want a different error message for that one? "Authentication page not found: Check your internet connection"?

Reply:
Yes, I think "Authentication Error: Please check your internet connection?".